client/my-sites/email/email-providers-stacked-comparison/provider-cards/provider-card-props.ts

```diff
@@ -17,6 +18,7 @@ export interface ProviderCard {
 	footerBadge?: ReactElement | null;
 	formFields?: ReactElement;
 	logo?: ReactElement | { path: string; className?: string };
+	logoFeatures?: LogoFeature[];
```

Comment:
I think we could keep the same name than in the old `Email Comparison` page, i.e. `appLogos`. `logoFeatures` doesn't sound right to me given that this is a list of logos, and not a list of features.